go.mod

```diff
@@ -17,6 +17,6 @@ require (
 	k8s.io/klog/v2 v2.6.0 // indirect
 )
 
-replace github.com/opencontainers/runtime-spec => github.com/kinvolk/runtime-spec v1.0.2-0.20201110202115-2755fc508653
+replace github.com/opencontainers/runtime-spec => github.com/kinvolk/runtime-spec v1.0.2-0.20210309175439-58798e75e980
```

Comment:
You could update this to some commit after https://github.com/opencontainers/runtime-spec/pull/1096 and remove the 'replace' directive, now that it is merged in runtime-spec upstream.

Reply:
Updated to that now! :)